corehq/ex-submodules/casexml/apps/case/models.py

```diff
@@ -178,8 +178,9 @@ class CommCareCase(DeferredBlobMixin, SafeSaveDocument, IndexHoldingMixIn,
     version = StringProperty()
     indices = SchemaListProperty(CommCareCaseIndex)
     case_attachments = SchemaDictProperty(CommCareCaseAttachment)
-    
+
     server_modified_on = DateTimeProperty()
+    _blobdb_type_code = CODES._default
```

Comment:
If this is only used for cases might as well call it `_case`? Or are there other places you plan to use it?